src/analysis/normed_space/star/gelfand_duality.lean

```diff
@@ -167,3 +169,46 @@ begin
 end
 
 end complex_cstar_algebra
+
+section functoriality
+
+namespace weak_dual
+
+namespace character_space
+
+variables {A B C : Type*}
+variables [normed_ring A] [normed_algebra ℂ A] [complete_space A]
+variables [star_ring A] [cstar_ring A] [nontrivial A]
```

Comment:
```suggestion
variables [star_ring A] [norm_one_class A]
```
And same for `B` and `C`

Reply:
Actually, I think if I merge master we won't even need this due to #16891